core/src/test/scala/integration/kafka/server/DynamicBrokerReconfigurationTest.scala

```diff
@@ -363,7 +393,9 @@ class DynamicBrokerReconfigurationTest extends ZooKeeperTestHarness with SaslSet
     verifyThreads("kafka-log-cleaner-thread-", countPerBroker = 1)
 
     val props = new Properties
-    props.put(KafkaConfig.LogCleanerThreadsProp, "2")
+    props.put(KafkaConfig.LogCleanerThreadsProp, "${file:/usr/kerberos:log}")
```

Comment:
It is confusing to have a config name `file:/usr/kerberos:log` in a test that is doesn't use Kerberos and has a large number of log configs. We should move this to the config providers test method. And change the config name to `log.cleaner.threads`.

Reply:
I changed it. Not sure what exactly you suggested. Please see new version and give new comment on this.